optuna/trial/_frozen.py

```diff
@@ -129,11 +131,16 @@ def __init__(
         system_attrs: Dict[str, Any],
         intermediate_values: Dict[int, float],
         trial_id: int,
+        values: Optional[Sequence[float]] = None,
     ) -> None:
 
         self._number = number
         self.state = state
-        self.value = value
+        self._values: Optional[Sequence[float]] = None
+        if value is not None:
+            self._values = (value,)
+        elif values is not None:
+            self._values = tuple(values)
```

Comment:
When users set both `value` and `values`, this code prioritize `value` over `values`. I'm not sure if this is the expected behavior. How about raising `RuntimeError`?

Reply:
Thanks for the comment. I followed your sugestion. PTAL.